SECURITY.md

```diff
@@ -10,4 +10,12 @@ Reported vulnerabilities will be investigated and patched on the next patch (or
 
 To report a vulnerability or a security-related issue, please email the private mailing list [cncf-strimzi-maintainers@lists.cncf.io](mailto:cncf-strimzi-maintainers@lists.cncf.io) with the details of the vulnerability.
 Do not report non-security-impacting issues through this channel.
-Use GitHub issues instead.
\ No newline at end of file
+Use GitHub issues instead.
+
+## Supported Versions
+
+Depending on the severity of the CVE or other vulnerability, on any Strimzi component, the project provides the fix either as a patch release of the current minor release or in the next minor release.
+For example, if the latest version of the cluster operator is 0.28.0, a vulnerability fix can be part of a 0.28.1 and/or 0.29.0.
```

Comment:
I would leave this out or use some _artificial_ versions. Using the current versions as example is IMHO confusing.

Reply:
I think that using an "artificial" version like X.Y.0 (if it's what you are referring to) is much more cryptic and confusing. I would take the example because it's always much better than 100 words and makes any explanation clearer.
I know that it's using a specific version, but it's just an example which doesn't need to be updated overtime.

Reply:
Artificial version could be also something like 99.99.0 - which does not conflict (and hopefully will not in the future) with any real version.